docs/sources/variables/chained-variables.md

```diff
@@ -0,0 +1,174 @@
+
+#### datacenter variable
+
+The query for this variable basically says, "Give me all the data centers that exist."
+
+```
+SHOW TAG VALUES  WITH KEY = "datacenter" 
+```
+
+The values returned are `America`, `Africa`, `Asia`, and `Europe`.
+
+#### host variable
+
+The query for this variable basically says, "Give me all hosts for the currently chosen data center." 
+
+```
+SHOW TAG VALUES WITH KEY = "hostname" WHERE "datacenter" =~ /^$datacenter$/
+```
+
+By itself, this query is nonsense, because `$datacenter$` has no value. However, if the user selects `America`, then the query changes to:
+
+```
+SHOW TAG VALUES WITH KEY = "hostname" WHERE "datacenter" =~ /^America/
+```
+
+The query returns all servers associated with `America`, including `server1`, `server2`, and so on.
+
+If the user selects `Europe`, then the query changes to:
+
+```
+SHOW TAG VALUES WITH KEY = "hostname" WHERE "datacenter" =~ /^Europe/
+```
+
+The query returns all servers associated with `Europe`, including `server3`, `server4`, and so on.
+
+#### More variables
+
+> **Note:** This example is theoretical. The InfluxDB server used in the example does not contain CPU metrics.
+
+The dashboard stops at two levels, but you could keep going. For example, if you wanted to get CPU metrics for selected hosts, you could copy the `host` variable and extend the query so that it reads:
+
+```
+SHOW TAG VALUES WITH KEY = "cpu" WHERE "datacenter" =~ /^$datacenter$/ AND "host" =~ /^$host$/
+```
+
+This query basically says, "Show me the CPU metrics for the selected host."
+
+Depending on what variable options the user selects, you could get queries like:
+
+```bash
+SHOW TAG VALUES WITH KEY = "cpu" WHERE "datacenter" =~ /^America/ AND "host" =~ /^server2/
+SHOW TAG VALUES WITH KEY = "cpu" WHERE "datacenter" =~ /^Africa/ AND "host" =~ /^server/7/
+SHOW TAG VALUES WITH KEY = "cpu" WHERE "datacenter" =~ /^Europe/ AND "host" =~ /^server3+server4/
+```
+
+## Best practices and tips
+
+The following practices will make your dashboards and variables easier to use.
+
+### Creating new linked variables
+
+- Chaining variables create parent/child dependencies. You can envision them as a ladder or a tree.
+- The easiest way to create a new chained variable is to copy the variable that you want to base the new one on. In the variable list, click the **Duplicate variable** icon to the right of the variable entry to create a copy. You can then add on to the query for the parent variable.
+- New variables created this way appear at the bottom of the list. You might need to drag it to a different position in the list to get it into a logical order.
+
+### Variable order
+
+You can change the orders of variables in the dashboard variable list by clicking the up and down arrows on the right side of each entry. Grafana lists variable dropdowns left to right according to this list, with the variable at the top on the far left.
+
+- List variables that do not have dependencies at the top.
```

Comment:
Personally this depends, if I have a variable that I'm not changing that often like interval, I would put that at the end because I would like to have the drop downs I use the most starting at the left in dashboard.

Reply:
I based this on recommendations from @torkelo . This list of best practices is specifically for chained variables, but this would be a great suggestion for a more general list of variable best practices. Got any others? :)